src/app/services/app-error.handler.ts

```diff
@@ -0,0 +1,38 @@
+import { ErrorHandler, Inject, Injector, Injectable } from '@angular/core';
+import { ToastrService } from 'ngx-toastr';
+import {TranslateService} from '@ngx-translate/core';
+
+@Injectable()
+export class AppErrorHandler implements ErrorHandler {
+
+  constructor(private i18n:TranslateService, @Inject(Injector) private readonly injector: Injector) {
```

Comment:
Shouldn't the ToastrService be injected in the constructor or the translationService retrieved from the injector ?